torchtitan/experiments/deepseek_v3/model_config.py

```diff
@@ -0,0 +1,183 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
nit: model_args.py? since ModelArgs

Reply:
Hmm, I think both work. My purpose of creating this file is to store configs of different DeepSeek flavors, like the `DeepSeek-V2-Lite` in it.